src/Files.App/Utils/Cloud/Detector/SyncCloudDetector.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System.IO;
+using Windows.Storage;
+
+namespace Files.App.Utils.Cloud
+{
+	/// <summary>
+	/// Provides a utility for Sync Cloud detection.
+	/// </summary>
+	public sealed class SyncCloudDetector : AbstractCloudDetector
+	{
+		protected override async IAsyncEnumerable<ICloudProvider> GetProviders()
+		{
+			string syncFolderPath = Path.Combine(Constants.UserEnvironmentPaths.HomePath, "Sync");
```

Comment:
If you change the path in the installer, this would not work.

I think it would be better to modify the [DetectGenericCloudDrive](https://github.com/files-community/Files/blob/main/src/Files.App/Utils/Cloud/CloudDrivesDetector.cs#L57) method. This method detects the existing cloud drives from the registry. I think that only small changes need to be made so that the SyncDrive is also recognized.

![image](https://github.com/user-attachments/assets/06185ea4-60de-4d9d-9c37-57672f49c3d3)

![image](https://github.com/user-attachments/assets/77fb699d-dbe9-45c6-bfdb-bb100db6ec40)

![image](https://github.com/user-attachments/assets/d8f92de6-6f70-4913-847b-695c113d5bf4)

![image](https://github.com/user-attachments/assets/c2392489-40c6-4f0f-bbfa-9a4897b79520)